packages/slice-machine/src/legacy/lib/models/common/widgets/Link/components.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Icon, Tooltip } from "@prismicio/editor-ui";
+import React, { useEffect, useState } from "react";
+import { Checkbox, Flex, Label } from "theme-ui";
+
+import { Col } from "@/legacy/components/Flex";
+
+type DisplayTextModel = { type: string };
+
+interface DisplayTextCheckboxProps {
+  text?: DisplayTextModel;
+  height?: 130 | 127;
+  setFieldValue: (
+    a: string,
+    b?: DisplayTextModel,
+    // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents, @typescript-eslint/no-explicit-any
+  ) => void | Promise<void | any>;
```

Comment:
💡 Small abbreviation

```suggestion
  ) => void | Promise<any>;
```

Reply:
💡 Maybe even like this, but I'm not sure if you want to enforce `void` if it's not a promise

```suggestion
  ) => any | Promise<any>;
```

Reply:
Good call, thanks. Fixed ✅